lib/services/api/survey/params/survey_info_params.dart

```diff
@@ -0,0 +1,14 @@
+part of '../survey_api_service.dart';
+
+class SurveyInfoParams extends ApiParams {
+  final String id;
+
+  SurveyInfoParams({
+    required this.id,
+  });
+
+  @override
+  void mapping(Mapper map) {
+    map("id", id, (v) {});
+  }
```

Comment:
redundant? Since you do not map anything 😅

Reply:
@manh-t cause `id` is a `final` so it can't be set from `mapping`
Therefore `SurveyInfoParams` is an immutable object 😁